doc/setup_instructions.md

```diff
@@ -551,6 +560,15 @@ is available through Homebrew.
 `msgmerge` is part of the gettext tools. So any errors with that
 missing are about these gettext tools.
 
+## Other tools
```

Comment:
Aren't these purely for native libraries, so this should be in the native libraries section? And llvm unless I very misremember how I setup the cmake files, it doesn't default to clang so g++ being installed is just fine with the gold linker.